.buildkite/commands/run-ui-tests.sh

```diff
@@ -20,8 +20,8 @@ tar -xf build-products.tar
 echo "--- :rubygems: Setting up Gems"
 install_gems
 
-echo "--- :cocoapods: Setting up Pods"
-install_cocoapods
```

Comment:
We can skip the pods because this step doesn't build the app. Unfortunately, the Swift Packages are still necessary because of how Fastlane calls `xcodebuild` under the hood (I think).